base/docs/basedocs.jl

```diff
@@ -1058,7 +1058,7 @@ f = open("file")
 try
     operate_on_file(f)
 catch
-    println("Zoinks!  An error occurred!)
+    println("Zoinks!  An error occurred!")
```

Comment:
"Zoinks!" may be a bit of humor, but is it necessary for docstrings?

Reply:
Yes.  Yes it is.

Reply:
Just to be clear, I am not saying that humor is not needed.
The example above this uses `@warn` to display a simple message. Here, `println` is used to tease.
As a humor deficient person, I think it is better not to require specific cultural knowledge or meta-context to the reader.